src/components/Patient/PatientHome.tsx

```
@@ -184,6 +186,17 @@ export const PatientHome = (props: {
       }}
       backUrl={facilityId ? `/facility/${facilityId}/patients` : "/patients"}
+      options={
+        <>
+          <Button asChild variant="primary">
+            <Link
+              href={`/facility/${facilityId}/patient/${id}/appointments/create`}
+            >
+              Doctor Consultation
+            </Link>
+          </Button>
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)